src/Ombi/ClientApp/app/issues/issueDetails.component.ts

```diff
@@ -85,4 +91,24 @@ export class IssueDetailsComponent implements OnInit {
     private loadComments() {
         this.issueService.getComments(this.issueId).subscribe(x => this.comments = x);
     }
+
+    private setBackground(issue: any) {
+        if (issue.requestType === 1) {
+            this.searchService.getMovieInformation(Number(issue.providerId)).subscribe(x => {
+                this.backgroundPath = this.sanitizer.bypassSecurityTrustStyle
+                    ("url(" + "https://image.tmdb.org/t/p/w1280" + x.backdropPath + ")");
+                this.posterPath = "https://image.tmdb.org/t/p/w300/" + x.posterPath;
+            });
+            
+        } else {
+            this.imageService.getTvBanner(Number(issue.providerId)).subscribe(x => {
+                this.backgroundPath = this.sanitizer.bypassSecurityTrustStyle
+                    ("url(" + x + ")");
+            });
+            this.searchService.getShowInformationTreeNode(Number(issue.providerId)).subscribe(x => {
```

Comment:
We probably shouldn't be doing this just to get the banner.
This will pull back ALL the information about the request, and we need to keep in mind that there might not be a request associated with the issue.

We can raise issues on content that do not have requests.

Reply:
@tidusjar Good point, I did not think of that. Would writing a few API calls that just returns image paths be the best solution here instead? For example, I wrote API calls inside image.service.ts but the FanArt banner and poster images were inconsistent with the request or search images. So maybe just writing new API calls for TheMovieDBApi.cs and TvMazeApi.cs to return just the images would work?

Reply:
Yes, we have the provider id so we can just use the Image service (check the Image Controller to see what we currently support)

If you are writing a new api then I suggest it should be on the ImageController.